Magento_Catalog/templates/product/list/items.phtml

```diff
@@ -0,0 +1,308 @@
+                                                <?php
+                                                $postDataHelper = $this->helper('Magento\Framework\Data\Helper\PostHelper');
+                                                $postData = $postDataHelper->getPostData($block->getAddToCartUrl($_item), ['product' => $_item->getEntityId()])
+                                                ?>
+                                                <button
+                                                    class="
+                                                        button
+                                                        catalog-grid-item__primary-action
+                                                        tocart
+                                                        primary
+                                                    "
+                                                    data-post='<?= /* @escapeNotVerified */ $postData ?>'
+                                                    type="button"
+                                                    title="<?= /* @escapeNotVerified */ __('Add to Cart') ?>"
+                                                >
+                                                    <?= /* @escapeNotVerified */ __('Add to Cart') ?>
+                                                </button>
+                                            <?php endif; ?>
+                                        <?php else: ?>
+                                            <?php if ($_item->getIsSalable()): ?>
+                                                <div class="stock available">
+                                                    <?= /* @escapeNotVerified */ __('In stock') ?>
+                                                </div>
+                                            <?php else: ?>
+                                                <div class="stock unavailable">
+                                                    <?= /* @escapeNotVerified */ __('Out of stock') ?>
+                                                </div>
+                                            <?php endif; ?>
+                                        <?php endif; ?>
+                                    <?php endif; ?>
+
+                                    <?php if ($showAddTo): ?>
+                                        <?php if ($addToBlock = $block->getChildBlock('addto')): ?>
+                                            <?= $addToBlock->setProduct($_item)->getChildHtml() ?>
+                                        <?php endif; ?>
+                                    <?php endif; ?>
+                                </div>
+                            <?php endif; ?>
+                        </section>
+                    </div>
+                    <?= ($iterator == count($items)+1) ? '</div>' : '' ?>
+                <?php endforeach ?>
+            </div>
+        </div>
+    </div>
+</div>
```

Comment:
wrong indentation or useless close div tag

Reply:
it was bad formatting